dali/test/python/test_utils.py

```diff
@@ -93,6 +93,8 @@ def check_batch(batch1, batch2, batch_size, eps = 1e-07):
                 diff1 = np.abs(batch1.at(i) - batch2.at(i))
                 diff2 = np.abs(batch2.at(i) - batch1.at(i))
                 err = np.mean( np.minimum(diff2, diff1) )
+                print("val1: {}\tval2: {}\tdiff: {}\teps: {}".format(batch1.at(i), batch2.at(i),
```

Comment:
remove

Reply:
I added it on purpose - I don't see good reason, why such logging data shouldn't be added to our tests, since it gives you better understanding what's happening inside